javaScript/readme.md

```diff
@@ -0,0 +1,6 @@
+# Code Challenges
```

Comment:
This file should be `README.md` uppercase.

Maybe not in this PR, but consider in a follow up PR, creating a root level README.md that describes the purpose of this repo and links to the 2 other readme one for Python and one for JS.

Then these Readme shouldn't have an upper heading # Code Challenges, but rather # JavaScipt Code Challenges